notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1882 @@
+{
```

Comment:
<p>Here as well, I suggest using the TiDE definition from the public doc: https://cloud.google.com/vertex-ai/docs/tabular-data/forecasting/train-model#training-methods</p><p> </p><p>"skip evaluation" is also a customization, right?</p><p>The following code demonstrates how you can create a forecasting pipeline with the following customizations:</p><ul><li>Skip evaluation</li><li>Limit the hyperparameter...</li></ul><p> </p><p>Can you add some comments in the code to show where the customizations are taking place? Which parameters are responsible for the setting the machine type, etc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Nice catch, thanks!</p>

Reply:
<p>Looks much better, but think this point is still unresolved:</p><p>"Can you add some comments in the code to show where the customizations are taking place? Which parameters are responsible for the setting the machine type, etc?"</p>

Reply:
<p>Added the comments.</p>